setup.py

```diff
@@ -27,12 +30,23 @@ def no_cythonize(extensions, **_ignore):
         extension.sources[:] = sources
     return extensions
 
+define_macros=[("QOI_MALLOC", "PyMem_RawMalloc"), ("QOI_FREE", "PyMem_RawFree"), ("NPY_NO_DEPRECATED_API", "NPY_1_7_API_VERSION")]
+if (
+    sys.version_info > (3, 13, 0)

```

Comment:
Since this is using python 3.13 could you update the build pipeline to test 3.13? We currently only test up to 3.12. https://github.com/kodonnell/qoi/blob/eedb9cc5c3ef763b6f0e2a64d9ab19143fe9815b/pyproject.toml#L26

Reply:
Ok, I'll have a try later. I build it on my pc and it works fine.